database/postgresql/src/main/java/com/example/javaopencommerce/order/OrderQueryRepositoryImpl.java

```diff
@@ -2,8 +2,10 @@
 
 import com.example.javaopencommerce.order.dtos.OrderDto;
 
+import javax.enterprise.context.ApplicationScoped;
 import java.util.UUID;
 
+@ApplicationScoped
```

Comment:
Not a fan of that annotation. It would be better to create bean with a @Producer.